### PR TITLE
Target Android 14 (SDK 34)

### LINKED ISSUE #3261: Jamlus not supporting android target version 34
I checked manifest file of android it's supporting following version     <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="28"/>

But currently Android 34 is launched in market so please migrate project for targetSdkVersion =34

### PR DESCRIPTION
**Pull request by npars**
**Short description of changes**

Update Android build to support Android 14.

CHANGELOG: Update Android build to support QT6 and latest dependencies supported by QT 6.7.2

**Context: Fixes an issue?**

Fixes #3261

**Does this change need documentation? What needs to be documented and how?**

**Status of this Pull Request**

**What is missing until this pull request can be merged?**

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [ ] I've filled all the content above

AUTOBUILD: Please build all targets

### CODE REVIEW

**Comment by ann0see** [comment]
Cool. Thank you very much!

File: src/main.cpp [hunk 1]
```diff
@@ -40,7 +40,7 @@
 #endif
 #include "util.h"
 #ifdef ANDROID
-#    include <QtAndroidExtras/QtAndroid>
+#    include <QtCore/private/qandroidextras_p.h>
```
Comment (ann0see) [inline src/main.cpp:43]: Does this break the qt5 build?
Is using "private" the best the way?

Comment (ann0see) [reply src/main.cpp:43]: https://github.com/jamulussoftware/jamulus/pull/3392 has a similar issue...

**Comment by ann0see** [comment]
The CI fails.

**Comment by pljones** [comment]
Can I propose we close this as encompassed by #3392, which we'll take into Jamulus 4?

**Comment by npars** [comment]
> Can I propose we close this as encompassed by #3392, which we'll take into Jamulus 4?

I would have no issue with that

**Comment by pljones** [comment]
Thanks!

**Pull request closed without merging**